crates/bevy_time/src/lib.rs

```diff
@@ -69,23 +71,27 @@ impl Plugin for TimePlugin {
 /// you may prefer to set the next [`Time`] value manually.
 #[derive(Resource, Default)]
 pub enum TimeUpdateStrategy {
```

Comment:
Does this affect `FixedTime`? Should it be mentioned here?

From my understanding `FixedTime` is updated from `Time`, maybe mentioning that briefly here and/or in https://docs.rs/bevy_time/0.11.0/bevy_time/fixed_timestep/index.html would be useful?